.github/workflows/ci.yml

```diff
@@ -58,10 +58,34 @@ jobs:
         set GeneratorPlatform=${{matrix.platform}}
         if ${{matrix.qtversion}}==Qt6 set UseQt6=ON
         if ${{matrix.platform}}==x86 set GeneratorPlatform=Win32
-        cmake -B ${{github.workspace}}/build -DCMAKE_TOOLCHAIN_FILE=c:/vcpkg/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=${{matrix.platform}}-windows -DCMAKE_GENERATOR_PLATFORM=%GeneratorPlatform% -DENABLE_SPICE=ON -DENABLE_TOOLS=ON -DENABLE_TESTS=ON -DENABLE_SDL=OFF -DENABLE_QT=ON -DUSE_QT6=%UseQt6% -DENABLE_FFMPEG=ON -DENABLE_MINIAUDIO=ON
+        cmake -B ${{github.workspace}}/build -DCMAKE_TOOLCHAIN_FILE=c:/vcpkg/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=${{matrix.platform}}-windows -DCMAKE_GENERATOR_PLATFORM=%GeneratorPlatform% -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/output -DENABLE_SPICE=ON -DENABLE_TOOLS=ON -DENABLE_TESTS=ON -DENABLE_SDL=OFF -DENABLE_QT=ON -DUSE_QT6=%UseQt6% -DENABLE_FFMPEG=ON -DENABLE_MINIAUDIO=ON -DX_VCPKG_APPLOCAL_DEPS_INSTALL=ON
 
     - name: Build
-      run: cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}} -- /maxcpucount:2 /nologo
+      run: cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}} --target install -- /maxcpucount:2 /nologo
+
+    - name: 'Checkout data'
```

Comment:
i don't think it's a good idea to add data into the package, because celestia.iss will depend on data files and with breaking changes you will need to update  celestia.iss

Reply:
do you think we should have a separate installer for data?